cli/tests/environment-activate.bats

```diff
@@ -81,7 +81,7 @@ env_is_activated() {
   run "$FLOX_BIN" install -d "$PROJECT_DIR" hello;
   assert_success
   assert_output --partial "✅ 'hello' installed to environment"
-  SHELL=bash USER="$REAL_USER" run -0 expect -d "$TESTS_DIR/activate/hello.exp" "$PROJECT_DIR";
+  SHELL=bash USER="$REAL_USER" NO_COLOR=1 run -0 expect -d "$TESTS_DIR/activate/hello.exp" "$PROJECT_DIR";
```

Comment:
suggestion (non-blocking): set `NO_COLOR` in setup method

Reply:
I want color everywhere else (for aesthetics), just not on these particular invocations.